pkg/channel/distributed/controller/README.md

```diff
@@ -5,12 +5,11 @@ reconciler, etc.) based on the latest knative-eventing SharedMain reconciler
 framework and utilities.
 
 The controller is based against the KafkaChannel CRD and reconciles all such
-instances in the K8S Cluster. It actually consists of two reconcilers, one for
-watching "Kafka" Secrets (those in knative-eventing labelled
-`eventing-kafka.knative.dev/kafka-secret: "true"`) which creates the Receiver /
-Producer Deployment & Service, and another which is watching `KafkaChannel`
-resources which provisions the Kafka Topic and creates the Dispatcher / Consumer
-Deployment & Service.
+instances in the K8S Cluster, watching `KafkaChannel` resources and performing
+the following functions:
+ - Provision the Kafka Topic
+ - Create the Dispatcher / Consumer Deployment And Service
+ - Create the Receiver / Producer Deployment And Service
 
```

Comment:
nit-picking, but maybe a bit more detail... ?

```
 - Provision the Kafka Topic
 - Create the single Receiver (Kafka Producer) Deployment And Service
 - Create a unique  Dispatcher (Kafka Consumer) Deployment And Service
```

Reply:
Modified this a bit